frontend/src/modules/history/components/Translation.js

```diff
@@ -254,169 +228,177 @@ export class TranslationBase extends React.Component<InternalProps, State> {
-                        dir={ locale.direction }
-                        lang={ locale.code }
-                        data-script={ locale.script }
-                    >
-                        <TranslationProxy
-                            content={ translation.string }
-                            diffTarget={
-                                this.state.isDiffVisible ? activeTranslation.string : null
+                                // Reject Button
+                                canReject ?
+                                    <Localized
+                                        id='history-Translation--button-reject'
+                                        attrs={{ title: true }}
+                                    >
+                                        <button
+                                            className='reject fa'
+                                            title='Reject'
+                                            name='reject'
+                                            onClick={ this.handleStatusChange }
+                                            disabled={ this.props.isActionDisabled }
+                                        />
+                                    </Localized>
+                                    :
+                                    <Localized
+                                        id='history-Translation--button-not-rejected'
+                                        attrs={{ title: true }}
+                                    >
+                                        <button
+                                            className='reject fa'
+                                            title='Not rejected'
+                                            disabled
+                                        />
+                                    </Localized>
                             }
-                            format={ entity.format }
-                        />
-                    </p>
+                            </menu>
+                        </header>
+                        <p
+                            className={ this.state.isDiffVisible ? 'diff' : 'default' }
+                            dir={ locale.direction }
+                            lang={ locale.code }
+                            data-script={ locale.script }
+                        >
+                            <TranslationProxy
+                                content={ translation.string }
+                                diffTarget={
+                                    this.state.isDiffVisible ? activeTranslation.string : null
+                                }
+                                format={ entity.format }
+                            />
+                        </p>
+                    </div>
+                </div>
+            </Localized>
+                <div>
+                    <CommentsList comments={ translation.comments } />
```

Comment:
You can remove the surrounding `<div>` (we already have one inside the `CommentsList`) and dedent the `CommentsList` to be on the same level as `Localized`.